classes/Smarty/TemplateFinder.php

```diff
@@ -35,7 +35,7 @@ class TemplateFinderCore
     private $directories;
     private $extension;
     private $productListEntities = ['category', 'manufacturer', 'supplier'];
-    private $productListSearchEntities = ['search', 'price-drop', 'best-sale'];
+    private $productListSearchEntities = ['search', 'prices-drop', 'best-sales', 'new-products'];
```

Comment:
Changing these strings is a BC :thinking: 
What if someone already used `price-drop` or `best-sale`?
Should it be?
```suggestion
    private $productListSearchEntities = ['search', 'price-drop', 'best-sale', 'prices-drop', 'best-sales', 'new-products'];
```

Reply:
This is how it's named everywhere (plural).
But your suggestion is a nice improvement ! It'll be more tolerant and avoid errors 👌